website/docs/basics/blocks-intro.md

```diff
@@ -67,30 +83,110 @@ You can follow the instructions given in [this chapter](blocks-component-in-bloc
 
 ### Where can I define my global styles?
 
+You can follow the instructions given in [this chapter](writing-styles).
+
 ### Where can I define global typography for all my blocks/components?
 
 You can follow the instructions given in [this chapter](writing-styles).
 
 ### Can I make a component with WP_Query logic in it?
 
+Yes, you can, but think of the component as a dumb and simple piece of code that should not hold any business logic. A component should only be used as a view. If you need to write WP_Query logic, you can do that in a block and just pass the data to your component as props.
+
 ### Do I need to make components for all my blocks?
 
-### Will, this setup work with full site editing?
+No, it might look like that from the examples provided, but that is not the case. You should create components depending on your needs.
+
+Ask yourself, will I reuse this functionality anywhere else?
+
+- if the answer is **yes**, create a component.
+- if the answer is **no**, don't create a component.
+
+Our recommendation is not to burden yourself from the begging with the components. Start creating blocks and if you find yourself in need of code that you already wrote, just extract it in a component.
+
+### Will this setup works with full site editing?
+
+Yes, it will. We are constantly upgrading this documentation and our code to say that it will work with full site editing. This boilerplate was made for Infinum/Eightshift WordPress team, so se are constantly using it on our projects.
 
 ### Can I use core blocks with your setup?
 
-### How can I limit my blocks list?
+Yes, you can. We are working on an ability to override core block on our smart way, but until we make this work, you can use everything the normal way that is defined in the core documentation. 
```

Comment:
```suggestion
Yes, you can. We are working on the ability to override core block in our smart way, but until we make this work, you can use everything the normal way that is defined in the core documentation. 

We avoid using core blocks because they add different class naming and additional markup that makes it harder to style things as we want to.

 Also, they are prone to breaking changes every several months, so we prefer to write our own implementation. 
```